omero/sysadmins/server-performance.rst

```diff
@@ -3,7 +3,26 @@ Performance and monitoring
 
 Once you have your OMERO server running and secured, a second
 critical step will be tuning various configuration parameters
-in order to get optimal performance.
+in order to get optimal performance. Assorted timeouts can be
+found under :ref:`Performance<performance_configuration>` but
```

Comment:
I think the build might complain about the lack of a space in your ref link here

Reply:
Seems happy enough and my paragraph is all nice and lined up on the right margin :smile:

Reply:
Hmm, Travis seems happy, checking the merge build now